app/views/layouts/application.html.erb

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HelloRailsReact</title>
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <%= csrf_meta_tags %>
+    <%= csp_meta_tag %>
+
+    <%= stylesheet_link_tag "application", "data-turbo-track": "reload" %>
+    <%= javascript_include_tag 'application', "data-turbo-track": "reload", defer: true %>
```

Comment:
- [ ] After I installed all gems and packages and try to precompile your app assets there is an error raised here in this image:

![image](https://user-images.githubusercontent.com/68796232/235509837-2c79f6a3-41f1-49ab-bbe9-0a5c9e12739e.png)

and that's happening because webpack is not installed and does not exist in packages.
After I installed webpack there is another error raised:

![image](https://user-images.githubusercontent.com/68796232/235510481-65812687-5ea7-44e6-86f7-0b9d0628781d.png)

Please fix these issues because your app now not precompile the javascript assets needed to run the react app.